src/App.js

```diff
@@ -14,6 +15,9 @@ function App() {
         <Route path="/login">
           <Login />
         </Route>
+        <Route path="/login/pwReset">
```

Comment:
by default, Routes match any URL that starts with the route path, and the first matched route is rendered. so /login/pwReset matches /login and the Login component is rendered. Suggest using a unique route like "/resetpassword".